### DISCUSSION TITLE
Allow android app to view all actions for a repo

### DISCUSSION THREAD

**Discussion by jasonwinterpixel** [community/community#50413] [Actions]
I want to see all the actions for a repo so I can see which ones have succeeded and failed.

We don't use pull requests, so the only way I can navigate to a failed action run is by clicking the link in the email.

I am on android..

EDIT
I selected actions as the category because I thought I was submitting feedback specifically about the mobile app, not about GitHub itself. That's another UX failure of the mobile app, I'd say. I pushed submit feedback inside the mobile app, the category should default to the mobile app. Actions is right at the top, so I blind clicked without understanding what I was doing.

Can a mod change this category to the mobile app?

**Comment by naimur-reza** [comment]
Unfortunately, the GitHub mobile app does not currently provide a way to view a list of all actions for a repository. You will need to use the GitHub website to access this information.

To view the list of all actions for a repository on the GitHub website:

1. Navigate to the repository in question.
2. Click on the "Actions" tab near the top of the page.
3. On the "Actions" page, you should see a list of all the workflows that have been run in the repository. You can click on any workflow to see a list of its runs.
4. On the workflow runs page, you can see a list of all the individual runs for that workflow, including their status (success, failure, etc.). 

To make it easier to find failed runs, you can use the search bar on the Actions page to search for failed or successful workflows. You can also use filters to narrow down the list of workflows and runs based on other criteria.
